modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase

```diff
@@ -379,21 +407,53 @@ definition {
 	}
 
 	@description = "This is a test for LPS-141370. View a Blog Collection in Display Page Template of Blogs"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ViewBlogCollectioninDPTofBlogs {
+		task ("Go to Masterclass Site") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-141370.
+		task ("Go to Display Page Templates") {
+			ProductMenu.gotoPortlet(
+				category = "Design",
+				portlet = "Page Templates");
+
+			NavItem.gotoPageTemplates();
+
+			Staging.gotoNavTab(tabName = "Display Page Templates");
+		}
+
+		task ("Go to Blog Entry and Assert the List of Blogs Presence") {
+			Click(
+				key_assetType = "Blog Entry",
+				locator1 = "Sharing#SHARED_CONTENT_ASSET_TYPE");
+
+			PageEditor.openPreviewMode();
+
+			AssertElementPresent(
+				key_colummnsNumber = "3",
+				locator1 = "Masterclass#COLUMNS_NUMBER");
+		}
 
 	}
 
 	@description = "This is a test for LPS-141370. View a List of Blogs in the Blog Detail View"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test ViewBlogListinBlogDetailView {
+		task ("Go to Masterclass Site") {
+			ApplicationsMenu.gotoSite(site = "Test Masterclass Name");
+		}
 
-		// TODO Create the automation about the LPS-141370.
+		task ("Go to Blog and Assert the List of Blogs Presence") {
+			Navigator._gotoPage(pageName = "Blog");
```

Comment:
Macros with _ should only be used inside the files where they are defined. Try to find another similar method in Navigator